phalcon/cache/frontend/msgpack.zep

```diff
@@ -0,0 +1,109 @@
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2016 Phalcon Team (https://phalconphp.com)       |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file docs/LICENSE.txt.                        |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ |          Vladimir Kolesnikov <vladimir@extrememember.com>              |
+ +------------------------------------------------------------------------+
+ */
+
+namespace Phalcon\Cache\Frontend;
+
+use Phalcon\Cache\Frontend\Data;
```

Comment:
@dred86 

It seems to me
```php
use Phalcon\Cache\Frontend\Data;
```
not needed here because class used the same namespace `Phalcon\Cache\Frontend`